toc/rfc/README.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Foundry Request For Comments
+
+Cloud Foundry community members use this directory as a forum to make, discuss and record both process and technical decisions. The outcomes of these discussions can be either an Action Plan, a new Standard or Process that the Cloud Foundry community should follow. This repository is open as a reference for all working groups and for the wider community.
+
+## Process
+
+1. Create a new branch on this repo and copy `rfc-000-template.md` to `rfc-000-my-proposal.md` and edit.
+2. If your RFC is only relevant for a single Working Group, please place it in a directory for the working group.
+   1. If your RFC is only relevant for a sigle Working Group, and the Working Group hosts RFCs elsewhere, please create a directory and place a link to where the RFCs can be found to aid in discoverability and follow the already agreed process. 
```

Comment:
Most of the time, the directory will already exist and should already point to the working group's RFC process. Maybe, we'll just have working groups file an RFC here that states the working group has a separate RFC process and provides a link to it. WDYT?
```suggestion
```

Reply:
How about 

```
If your RFC is only relevant for a single Working Group, and the Working Group hosts RFCs elsewhere and this step has not already been completed, please create a directory and place a link to where the RFCs can be found to aid in discoverability and follow the already agreed process.
```